src/telemetry.ts

```diff
@@ -3288,10 +3422,27 @@ export class IEventNamePropertyMapping {
             }
         }
     };
+    /**
+     * Error information from the debugger output channel while running initialization code.
+     */
     [Telemetry.InteractiveWindowDebugSetupCodeFailure]: TelemetryEventInfo<{
         ename: string;
```

Comment:
You'll need to add comments to this.
Please merge main and run `npm run generateTelemetry`, the errors will be logged into the console window.
Or errors would be in TELEMETRY.md file.

Reply:
Telemetry.md file did not finish generating, and the error from that npm script isn't very helpful:

TypeError: Cannot read properties of undefined (reading 'comment')
    at C:\src\vscode-jupyter\out\telemetryGenerator.node.js:558:104
    at Array.forEach (<anonymous>)
    at visit (C:\src\vscode-jupyter\out\telemetryGenerator.node.js:501:26)
    at visitNodes (C:\src\vscode-jupyter\node_modules\typescript\lib\typescript.js:30672:30)
    at Object.forEachChild (C:\src\vscode-jupyter\node_modules\typescript\lib\typescript.js:30951:24)
    at generateDocumentation (C:\src\vscode-jupyter\out\telemetryGenerator.node.js:484:20)
    at generateTelemetryOutput (C:\src\vscode-jupyter\out\telemetryGenerator.node.js:732:5)
    at async Object.default_1 [as default] (C:\src\vscode-jupyter\out\telemetryGenerator.node.js:740:5)
    at async generateTelemetryMD (C:\src\vscode-jupyter\gulpfile.js:414:5)

Reply:
sorry i missed this, Let me check this

Reply:
Please try re-basing with main branch.
Change all of `measures: commonClassificationForDurationProperties` to `measures: commonClassificationForDurationProperties()`

Reply:
I managed to get this working by merely merging the main branch changes, see here https://github.com/microsoft/vscode-jupyter/pull/11239